index.html

```diff
@@ -0,0 +1,1383 @@
+                                    <img src="https://d.newsweek.com/en/full/1611327/cover-july-31-august-07-2020.jpg?w=170&q=95&f=e9849b39e78f364f0f8bb410954aa063" alt="hello" />
+                                </div>
+                        </div>
+                        <div class="col-sm-12 subscribe-sect col-md-7 col-lg-6 mt-md-5">
+                            <header class="text-center">
+                                <h3 class="text-md-left"><strong>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</strong></h3>
+                                <p>All Access + Weekly Delivery. Choose A Membership That's Perfect for You!</p>
+                            </header>
+                            <div class="f1 info w-100 border-danger justify-content-center text-center">
+                                <button type="button" class="btn btn-danger mt-2 mb-2 ml-2 mr-2">SUBSCRIBE ></button>
+                            </div>
+                        </div>
+                    </div>
+                </article>
+                <article class="newsletter col-sm-12 col-md-12 ml-lg-1 col-lg-4 mb-3 mb-md-0 mt-n4">
+                    <header class="block-title mx-0 mx-md-0 ml-lg-n3 mt-lg-4 mb-lg-3 text-danger pt-md-2"><strong>NEWSLETTER</strong></header>
+                    <div class="d-lg-flex justify-content-center border-lg-none" id="news-border">
+                        <div class="news-block row justify-content-center border mt-1 mt-lg-n1 mb-3">
+                            <div class="col-sm-3 col-lg-6 d-none d-md-block d-lg-block" id="newsletter-large-pic"></div>
+                            <div class="smalldiv col-sm-9 col-lg-6 w-100 pl-3 py-2 border mb-2">
+                                <div class="pt-md-3">
+                                    <h3 class="text-center text-lg-left mb-3 text-md-left"><strong>The Starting 5</strong></h3>
+                                    <p class="text-center text-md-left">See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+                                </div>
+                                <div class="mt-3 mb-n2">
+                                    <input type="text" class="form-text form-control" placeholder="Email address" />
+                                    <input type="submit" class="form-submit btn btn-primary btn-block btn-danger mt-2" value="FREE SIGN UP" />
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </article>
+            </div>
+        </div>
+        <footer class="container bg-white pt-5 pt-lg-0" id="sticky-footer">
+            <div class="d-flex flex-row pl-3">
+                <div><img src="images/newsweeklogo.png" id="footer-logo" alt="hello"/></div>
+                <span class="footer-copyright small ml-2 mr-auto mt-4 pr-md-2">© 2020 NEWSWEEK</span>
```

Comment:
- [ ] You are not using footer-copyright anymore in your CSS, please take it out

Reply:
Removed unused class